ci/build_windows.bat

```diff
@@ -22,6 +22,9 @@ if %errorlevel% neq 0 goto :error
 python -m pip install --upgrade pip setuptools wheel
 if %errorlevel% neq 0 goto :error
 
+pip install --upgrade --user awscli
```

Comment:
Can this be inlined with line 22? Given you also use `--user` here, maybe it's not possible.

Reply:
yep. Just did that. Hopefully it should work, otherwise we can do a `--user`

Reply:
this works: https://jenkins.int.snowflakecomputing.com/job/BuildPyConnector-Win/